Header header

float64 d_groundtruth
float64 d_raw_bb
float64 d_raw_alt
float64 bebop_alt
float64 beta
float64 sigma_1
float64 sigma_2
float64 cam_tilt
float64 target_height
float64 target_grounddist
uint16  bb_height

float64 vref_x
float64 vref_y
float64 vref_z
float64 vref_yaw

float64 vest_x
float64 vest_y

float64[6] task_err
